ch_7_GPU_Support/README.md

```diff
@@ -0,0 +1,175 @@
+# Remove latest kernel
+sudo apt purge linux-image-6.2.0-26-generic
+
+sudo apt autoremove
+
+sudo reboot
+
+# Install drivers
+sudo apt-get update
+
+sudo apt-get -y install \
+    gawk \
+    dkms \
+    linux-headers-$(uname -r) \
+    libc6-dev
+	
+sudo apt-get install -y intel-platform-vsec-dkms intel-platform-cse-dkms intel-i915-dkms intel-fw-gpu
+
+sudo apt-get install -y gawk libc6-dev udev\
+  intel-opencl-icd intel-level-zero-gpu level-zero \
+  intel-media-va-driver-non-free libmfx1 libmfxgen1 libvpl2 \
+  libegl-mesa0 libegl1-mesa libegl1-mesa-dev libgbm1 libgl1-mesa-dev libgl1-mesa-dri \
+  libglapi-mesa libgles2-mesa-dev libglx-mesa0 libigdgmm12 libxatracker2 mesa-va-drivers \
+  mesa-vdpau-drivers mesa-vulkan-drivers va-driver-all vainfo
+  
+sudo reboot
+
+# Configuring permissions
+sudo gpasswd -a ${USER} render
+
+newgrp render
+
+# Verify the device is working with i915 driver
+sudo apt-get install -y hwinfo
+hwinfo --display
+
+
+# Install one api
+wget -O- https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB | gpg --dearmor | sudo tee /usr/share/keyrings/oneapi-archive-keyring.gpg > /dev/null
+
+echo "deb [signed-by=/usr/share/keyrings/oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main" | sudo tee /etc/apt/sources.list.d/oneAPI.list
+
+sudo apt update
+
+sudo apt install intel-basekit
+```
+</details>
+
+### 7.0.3 Python Environment Setup
+
+Next, use a python environment management tool (we recommend using [Conda](https://docs.conda.io/projects/conda/en/stable/)) to create a python enviroment and install necessary libs.
+
+#### 7.0.3.1 Install Conda
+
+For Linux users, open a terminal and run below commands:
+
+```bash
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash ./Miniconda3-latest-Linux-x86_64.sh
+conda init
+```
+
+> **Note**
+> Follow the instructions popped up on the console until conda initialization finished successfully.
+
+#### 7.0.3.2 Create Environment
+
+> **Note**
+> Python 3.9 is recommended for running BigDL-LLM.
+
+Create a Python 3.9 environment with the name you choose, for example `llm-tutorial-gpu`:
+
+```bash
+conda create -n llm-tutorial-gpu python=3.9
+```
+
+Then activate the environment `llm-tutorial-gpu`:
+
+```bash
+conda activate llm-tutorial-gpu
+```
+
+### 7.0.4 Best Known Configuration on Linux
+
+For optimal performance on Intel GPUs, it is recommended to set several environment variables:
+
+```bash
+# configure OneAPI environment variables
+source /opt/intel/oneapi/setvars.sh
+
+export USE_XETLA=OFF
+export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+```
+
+### 7.0.5 Jupyter Service Setup
+
+#### 7.0.5.1 Install Jupyter
+
+The `jupyter` library is required for running the tutorial notebooks (i.e. the `.ipynb` files). Under your activated Python 3.9 environment, run:
+
+```bash
+pip install jupyter
+```
+
+#### 7.0.5.2 Start Jupyter Service
+
+Then start jupyter service through
+
+```bash
+jupyter notebook
```

Comment:
I am not sure what should be the best start up command for servers with Intel Data Center GPU Flex Series. May be we should add numactl for the loading process.